src/components/Footer/Footer.js

```diff
@@ -1,116 +1,7 @@
-// import React from "react";
```

Comment:
@madhunivi77 you never want to commit code that is commented out - with Github you have a version history so there's no need